src/sagemaker/mxnet/estimator.py

```diff
@@ -47,10 +48,12 @@ def __init__(self, entry_point, source_dir=None, hyperparameters=None, py_versio
                 to convert them before training.
             py_version (str): Python version you want to use for executing your model training code (default: 'py2').
                               One of 'py2' or 'py3'.
+            framework_version (str): MXNet version you want to use for executing your model training code.
```

Comment:
Can you add info to this docstring to tell users how they can figure out what valid values are? (Same for TF as well.)

Reply:
Yep! 👍